chrome/content/zotero/preferences/preferences_cite.xul

```diff
@@ -74,9 +75,13 @@
 					<groupbox>
 						<caption label="&zotero.preferences.citationOptions.caption;"/>
 						
+						<label value="&zotero.preferences.cite.bibliographyLocale;" control="bibliographyLocale-menu"/>
+						<menulist id="bibliographyLocale-menu"/>
```

Comment:
We should put this on the same line with the label (in an `<hbox>`) to save space.